src/utils/enums.ts

```diff
@@ -28,6 +28,7 @@ export enum NAME_VALIDATION_STATUS {
 export enum DB_COLLECTIONS {
   ADMINS = "admins",
   BANNED = "banned",
+  MUTED = "muted",
```

Comment:
Do we will use the same collection for ppl muted on vocal chat ? I know it's not ready but if we plan to separate chat and voice mute let's rename the collection and enum CHAT_MUTED

Reply:
we'll use the same collection I think